### ISSUE TITLE
Why don't the line numbers show by default?

### ISSUE THREAD

**Issue by asgeo1** [bugsnag/bugsnag-cocoa#84]
I'm a bit confused by Bugsnag, because it doesn't seem to show the line numbers in the crash logs. 

I was using Crashlytics previously, and it was able to show me the method names and line numbers for a crash. But I'm not getting the line numbers for Bugsnag in the crash reports.

Apparently I can upload the dsym to get this information.  I'm not really sure about that, because after I ran the suggested script, it still didn't show me any line numbers in the crash report.  (and it says "Still Missing" afterwards, which I don't get - maybe it's talking about other targets?).

Anyway why is it that out-of-the-box Crashlytics can show the line numbers, but Bugsnag can't? I don't have to upload any such dsym data manually for Crashlytics...

Or could there be something wrong with my Bugsnag configuration?  (I installed version 4.1.0 via Cocoapods...)

**Comment by kattrali** [comment]
Hi @asgeo1, thank you for the report. 

> Apparently I can upload the dsym to get this information. I'm not really sure about that, because after I ran the suggested script, it still didn't show me any line numbers in the crash report.

Correct. After uploading your dSYM file, we can translate the raw stacktrace into the method names and line numbers involved. You can configure your project to upload dSYM data automatically, though it should have happened automatically when installing Bugsnag via Cocoapods. Can you check your project builds phases for a "Run Script" phase named "Upload Bugsnag dSYM"? If its not there, you can add it manually and retry, using the [non-cocoapods installation guide, Step 5](https://bugsnag.com/docs/notifiers/cocoa#without-cocoapods). This is very similar to how Crashlytics handles symbolication as well. 

Was there any output from running the dSYM upload script?

**Comment by asgeo1** [comment]
Hi @kattrali,

You're right, there is no "Upload Bugsnag dSYM" run script.  That's strange, that it didn't get added for my project.  I've tried removing the pod and re-adding it, but it never gets added to my project.

Anyway, I followed the instructions to manually add the run script, and that has worked - I'm now seeing the line numbers in Bugsnag.


With regards to running the script manually on my machine, I was getting a `Still Missing` message output.  I'm not sure what that was about, but I guess it doesn't matter since it's working fine now.

Cheers!
